.gitpod.yml

```
@@ -0,0 +1,4 @@
+# Learn more about this file at https://www.gitpod.io/docs/references/gitpod-yml
+tasks:
+  - init: pnpm install
+  command: pnpm nr build --watch
```

```suggestion
tasks:
  - name: Setup & Build
    init: pnpm install
    command: pnpm nr build --watch
```

I have encountered YAML linters which were not happy with indentation like this, so I learned this the hard way, haha. It also makes sense to name tasks for easier understanding of what a task accomplishes